File: guitar-dashboard-for-ios/guitar-dashboard-for-ios/Models/Patch.swift
//
//  Patch.swift
//  guitar-dashboard-for-ios
//
//  Created by Guglielmo Frigerio on 28/12/21.
//

import Foundation

struct Patch: Decodable {
    let name: String
    let axeFx2: Program
    let axeFx3: Program
}

File: XmlToJsonConverter/XmlToJsonConverter/Models/Patch.cs
using System;
using System.Xml.Linq;

namespace XmlToJsonConverter.Models
{
	public class Patch
	{
		public Patch(XElement patchElement)
		{
			name = patchElement.Attribute("name")?.Value;
			axeFx3 = new Program(patchElement);
		}

		public string name { get; set; }

		public Program axeFx2 { get; set; }

		public Program axeFx3 { get; set; }
	}
}

